package.json

```diff
@@ -23,6 +23,7 @@
     "lib",
     "cli.js"
   ],
+  "typings": "./d.ts/index.d.ts",
```

Comment:
@hzlmn @jbucaran I believe all editors' typescript plugins _do_ pickup the `types` field. This is what allows all of them to read definitions & provide completions.